### DISCUSSION TITLE
Unable to View jupyter notebooks in app

### DISCUSSION THREAD

**Discussion by i-karen** [community/community#20615] [Mobile]
Can we add ipynb rendering functionality?

It works on desktop.

Would be super helpful if it works on app

**Comment by iskae** [comment]
Hi @i-karen 👋 

Thanks so much for reaching out and for your request ❤️ 

We already have an internal tracking issue for your request 🎉 I will be adding your comment in there too and hopefully we can support ipynb files in the near future 👍
